tests/everflow/everflow_test_utilities.py

```diff
@@ -744,15 +744,28 @@ def get_expected_mirror_packet(mirror_session, setup, duthost, mirror_packet, ch
         if duthost.facts["asic_type"] in ["barefoot", "cisco-8000"]:
             payload = binascii.unhexlify("0" * 24) + str(payload)
 
-        expected_packet = testutils.simple_gre_packet(
-            eth_src=setup["router_mac"],
-            ip_src=mirror_session["session_src_ip"],
-            ip_dst=mirror_session["session_dst_ip"],
-            ip_dscp=int(mirror_session["session_dscp"]),
-            ip_id=0,
-            ip_ttl=int(mirror_session["session_ttl"]),
-            inner_frame=payload
-        )
+        #Add vendor specific ttl check ,reused pkts in mirror session has 1 decrement in TTL 
```

Comment:
I think don't need a if-else branch here as most of the code is the same, just add one line code to minus the TTL after expected_packet is generated for Cisco-8000.

Reply:
Updated the changes .. please review